wagtail/admin/templates/wagtailadmin/pages/bulk_actions/confirm_bulk_delete.html

```diff
@@ -0,0 +1,33 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags %}
+{% block titletag %}{% blocktrans count counter=pages|length with title=page.get_admin_display_title  %}Delete 1 page {% plural %}Delete {{ counter }} pages{% endblocktrans %}{% endblock %}
+
+{% block content %}
+    {% trans "Delete" as del_str %}
+    {% include "wagtailadmin/shared/header.html" with title=del_str subtitle=page.get_admin_display_title icon="doc-empty-inverse" %}
+
+    <div class="nice-padding">
+        <p>{% blocktrans with snippet_type_name=model_opts.verbose_name_plural %}Are you sure you want to delete these pages?{% endblocktrans %}</p>
```

Comment:
This doesn't use the `with` variable

Reply:
@jacobtoppm `blocktrans` does use `with` in its syntax - https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#std:templatetag-blocktrans. Unless I misunderstood your comment

Reply:
Sorry, don't think I was clear enough. I know `with` is valid syntax, but `snippet_type_name` isn't actually being used inside the tag.